packages/@ourworldindata/grapher/src/stackedCharts/AbstractStackedChart.tsx

```diff
@@ -251,16 +250,16 @@ export class AbstractStackedChart
         return new AxisConfig(this.manager.yAxisConfig, this)
     }
 
+    // implemented in subclasses
+    @computed protected get yAxisDomain(): [number, number] {
```

Comment:
I find it confusing if methods are overwritten in subclasses, so I use a "dummy" implementation here, and it's then the job of a subclass to overwrite it. (We can't use actual abstract class because AbstractStackedChart cannot be made abstract because of MobX)

Reply:
yeah, I agree that this is often confusing...